.rubocop.yml

```diff
@@ -2,4 +2,4 @@ AllCops:
   NewCops: enable
   
 Metrics/BlockLength:
-  Max: 100
\ No newline at end of file
+  AllowedMethods: ["describe", "context"]
```

Comment:
Do you have some insight as to why this works? I spent hours searching for answers before submitting this without understanding it. The rubocop doc doesn't really have any answers I can find on this. I actually had this originally before changing to the Max: 100 because I at least understood how it worked. I had originally thought that inserting text into describe and context was meant to describe and give context to why you were allowing the method to pass, but if I insert any other text into those fields it fails. Hopefully that makes sense.

Reply:
I'm not sure I understand your meaning exactly, but basically line 4 is specifying the metric for block length and line 5 is specifying the names of methods that are allowed to bypass block length restrictions.  `describe` and `context` are the methods used in rspec and because they tend to be long when there are a lot of tests, we don't want them to be flagged.  Let me know if that doesn't make sense and I'll try to explain better tomorrow.

Reply:
That gives me everything I need to know. Thanks!!